frontend/src/test/scala/bloop/TestSpec.scala

```diff
@@ -91,10 +96,10 @@ object JsTestSpec extends BaseTestSpec("test-projectJS-test", "cross-test-build-
        |1 tests, 1 passed
        |All tests in hello.UTestTest passed
        |
-       |===============================================
+       |$delimiter
```

Comment:
This needs a string interpolation at the start of the multiline string, that's why the tests are failing

Reply:
Fixed.